tests/gtest/avifrgbtoyuvtest.cc

```diff
@@ -358,26 +358,46 @@ TEST(RGBToYUVTest, AllMatrixCoefficients) {
         for (avifRange yuv_range : {AVIF_RANGE_LIMITED, AVIF_RANGE_FULL}) {
           for (decltype(AVIF_MATRIX_COEFFICIENTS_IDENTITY) matrix_coefficients :
                {
-                   AVIF_MATRIX_COEFFICIENTS_BT709,
-                   AVIF_MATRIX_COEFFICIENTS_UNSPECIFIED,
-                   AVIF_MATRIX_COEFFICIENTS_FCC,
-                   AVIF_MATRIX_COEFFICIENTS_BT470BG,
-                   AVIF_MATRIX_COEFFICIENTS_BT601,
-                   AVIF_MATRIX_COEFFICIENTS_SMPTE240,
-                   AVIF_MATRIX_COEFFICIENTS_YCGCO,
-                   AVIF_MATRIX_COEFFICIENTS_BT2020_NCL,
-                   AVIF_MATRIX_COEFFICIENTS_CHROMA_DERIVED_NCL
-                   // These are unsupported. See avifPrepareReformatState().
-                   // AVIF_MATRIX_COEFFICIENTS_BT2020_CL
-                   // AVIF_MATRIX_COEFFICIENTS_SMPTE2085
-                   // AVIF_MATRIX_COEFFICIENTS_CHROMA_DERIVED_CL
-                   // AVIF_MATRIX_COEFFICIENTS_ICTCP
+                 AVIF_MATRIX_COEFFICIENTS_BT709,
+                     AVIF_MATRIX_COEFFICIENTS_UNSPECIFIED,
+                     AVIF_MATRIX_COEFFICIENTS_FCC,
+                     AVIF_MATRIX_COEFFICIENTS_BT470BG,
+                     AVIF_MATRIX_COEFFICIENTS_BT601,
+                     AVIF_MATRIX_COEFFICIENTS_SMPTE240,
+                     AVIF_MATRIX_COEFFICIENTS_YCGCO,
+                     AVIF_MATRIX_COEFFICIENTS_BT2020_NCL,
+                     AVIF_MATRIX_COEFFICIENTS_CHROMA_DERIVED_NCL
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+                     ,
+                     AVIF_MATRIX_COEFFICIENTS_YCGCO_RE,
+                     AVIF_MATRIX_COEFFICIENTS_YCGCO_RO
```

Comment:
Add a comma (,) to the end of this line. I think it is allowed in both C++ and C now. Does it not work?